File: packages/py/tracing/examples/openai_agents.py
from agents import Agent, Runner
from uselemma_tracing import instrument_openai_agents

instrument_openai_agents()

agent = Agent(
    name="support-agent",
    instructions="Answer customer questions clearly and concisely.",
)


async def call_agent(user_message: str):
    result = await Runner.run(agent, user_message)
    return result.final_output

File: packages/ts/tracing/examples/openai-agents.ts
import { Agent, addTraceProcessor, run } from "@openai/agents";
import { openAIAgents } from "@uselemma/tracing";

addTraceProcessor(openAIAgents());

const agent = new Agent({
  name: "support-agent",
  instructions: "Answer customer questions clearly and concisely.",
});

const result = await run(agent, "Where is my order?");

console.log(result.finalOutput);
